### PR TITLE
#239 made sure there wasn't an inversion, wrote a unit test to check …

### PR DESCRIPTION
**Pull request by evalott100**
…constants set appropriately to the NexusWriter

### CODE REVIEW

File: src/artemis/nexus_writing/write_nexus.py [hunk 1]
```diff
@@ -162,7 +162,7 @@ def create_detector_parameters(detector_params: DetectorParams) -> Dict:
         "flatfield_applied": "_dectris/flatfield_correction_applied",
         "pixel_mask": "mask",
         "pixel_mask_applied": "_dectris/pixel_mask_applied",
-        "image_size": [detector_pixels.width, detector_pixels.height],  # (fast, slow)
+        "image_size": [detector_pixels.height, detector_pixels.width],  # (fast, slow)
```
Comment (DominicOram) [inline src/artemis/nexus_writing/write_nexus.py:165]: Should: This comment is now wrong, it should be  `# (slow, fast)` to match the `fast_axis`, `slow_axis` above.

**Review by DominicOram — CHANGES_REQUESTED** [review]
Looks good to me, minor comment in code, otherwise good, thanks!

**Review by evalott100 — COMMENTED** [review]
Changes made

**Review by DominicOram — APPROVED** [review]
Great, thanks!

**Pull request merged**